src/components/NavBar.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] Kindly remove this & solve it manually.

Reply:
@Deepakdanger I have tried different ways but I was not able, that's why I opted for using eslint disabler